js/ui/map.js

```diff
@@ -548,11 +585,11 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
-     * Replaces the map's style object with a new value. Unlike the `style`
-     * option in the Map constructor, this method only accepts an object
-     * of a new style, not a URL string.
+     * Replaces the map's Mapbox GL style object with a new value.
+     *
+     * Unlike the `style` option in the `Map` constructor, this method only accepts an object, not a URL string.
```

Comment:
Let's remove this line. `setStyle` does accept URL strings as the [normalization is handled within `Style`](https://github.com/mapbox/mapbox-gl-js/blob/master/js/style/style.js#L71-L75).